layouts/partials/header-new.html

```diff
@@ -0,0 +1,99 @@
+	    <link rel="apple-touch-icon-precomposed" sizes="60x60" href="/assets/images/icons/apple-touch-icon-60x60.png" />
+	    <link rel="apple-touch-icon-precomposed" sizes="120x120" href="/assets/images/icons/apple-touch-icon-120x120.png" />
+	    <link rel="apple-touch-icon-precomposed" sizes="76x76" href="/assets/images/icons/apple-touch-icon-76x76.png" />
+	    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="/assets/images/icons/apple-touch-icon-152x152.png" />
+	    <link rel="icon" type="image/png" href="/assets/images/icons/favicon-196x196.png" sizes="196x196" />
+	    <link rel="icon" type="image/png" href="/assets/images/icons/favicon-96x96.png" sizes="96x96" />
+	    <link rel="icon" type="image/png" href="/assets/images/icons/favicon-32x32.png" sizes="32x32" />
+	    <link rel="icon" type="image/png" href="/assets/images/icons/favicon-16x16.png" sizes="16x16" />
+	    <link rel="icon" type="image/png" href="/assets/images/icons/favicon-128.png" sizes="128x128" />
+	    <meta name="application-name" content="&nbsp;" />
+	    <meta name="msapplication-TileColor" content="#212146" />
+	    <meta name="msapplication-TileImage" content="/assets/images/icons/mstile-144x144.png" />
+	    <meta name="msapplication-square70x70logo" content="/assets/images/icons/mstile-70x70.png" />
+	    <meta name="msapplication-square150x150logo" content="/assets/images/icons/mstile-150x150.png" />
+	    <meta name="msapplication-wide310x150logo" content="/assets/images/icons/mstile-310x150.png" />
+	    <meta name="msapplication-square310x310logo" content="/assets/images/icons/mstile-310x310.png" />
+       
+        <!-- OpenGraph Tags -->
+	    <meta property="og:site_name" content="{{ if .Title }}{{ .Title }}{{ else }}{{ .Site.Title }}{{ end }}" />
+	    <meta property="og:title" content="{{ if .Title }}{{ .Title }}{{ else }}{{ .Site.Title }}{{ end }}" />
+	    {{ if .Description }}<meta property="og:description" content="{{ .Description }}" />{{ end }}
+	    <meta property="og:url" content="{{ .Permalink }}" />
+	    <meta property="og:type" content="website" />
+        <!-- Page image -->
+        {{ if .Params.image -}}
+            <link rel="image_src" href="{{ .Site.BaseURL }}/assets/images/opengraph-images/{{ .Params.image }}" />
+            <meta property="og:image" content="{{ .Site.BaseURL }}/assets/images/opengraph-images/{{ .Params.image }}" />
+            <meta property="og:image:url" content="{{ .Site.BaseURL }}/assets/images/opengraph-images/{{ .Params.image }}" />
+            <meta property="og:image:secure_url" content="{{ .Site.BaseURL }}/assets/images/opengraph-images/{{ .Params.image }}" />
+        {{- end }}
+
+        <!-- OpenGraph and Twitter tags -->
+        {{ if .IsPage -}}
+            <meta property="og:type" content="article" />
+            <meta property="og:article:published_time" content="{{ .Date.Format "2006-01-02T15:04:05Z07:00" | safeHTML }}" />
```

Comment:
Rather not use a hardcoded date. Perhaps we remove that for now

Reply:
done